moto/ec2/responses/launch_templates.py

```diff
@@ -187,11 +187,7 @@ def describe_launch_template_versions(self) -> str:
         ret_versions = []
         if versions and template is not None:
             for v in versions:
-                if str(v).lower() == "$latest" or "$default":
```

Comment:
This if-statement always returns True, because `or "$default"` is always True.

[Originally](https://github.com/getmoto/moto/commit/2c47ee10b182e63865bc9b95674fafab38a3ff27) it obviously meant to say `if str(v).lower() in ["$latest", "$default"]`. However, because the `get_version` method already checks whether `v` is $latest/$default/numeric, the whole if-statement is pointless.

(`get_version` was introduced here: https://github.com/getmoto/moto/commit/cf2690ca1e2e23e6ea28defe3e05c198d9581f79)